packages/amplify-provider-awscloudformation/src/utils/auth-types.ts

```diff
@@ -9,10 +9,10 @@ export interface AuthFlowConfig extends Partial<AwsSdkConfig> {
 }
 
 export interface AwsSdkConfig {
-  accessKeyId: string;
+  accessKeyId?: string;
```

Comment:
why are these no longer required?

Reply:
The `amplify-provider-cloudformation` package was failing build without these changes for me. Probably something wrong with my setup. Reverted these changes.